index.html

```diff
@@ -469,6 +469,35 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ________ Farookh Contributor card END ________  -->
 
+
+        <!-- ________ *Jatin* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Jatin</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com" target="https://twitter.com/jtpandey">jtpandey</a>
+            <i class="fab fa-brands fa-github-alt"></i>
+            <a href="https://www.github.com" target="https://github.com/jt-pandey">jt-pandey</a>
+          </p>
+          <p class="about">I am a <b>newbie</b> who wants to learn more about contributing to Open Source Projects</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="https://roadmap.sh/" title="Road Map">Road Map</a>
```

Comment:
Could you please put the links for the three resources in the `href` tag, not the `target` tag? Will merge after these changes are made!

Thanks!